### PR TITLE
fix(addon-doc): use correct dependencies

### LINKED ISSUE #1274: 🐞 - "addon-doc" incompatible with Angular v13
### Which @taiga-ui/* package(s) are the source of the bug?

addon-doc

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/taiga-starter

### Description

Trying to update Angular to v13 with `ng update @angular/core@13 @angular/cli@13` throws following error when using `@taiga-ui/addon-doc`:
```
Package "@taiga-ui/addon-doc" has an incompatible peer dependency to "@angular-devkit/schematics" (requires "^9.0.0", would install "13.2.0").
```

While you are at it, it would be nice if all other `addon-doc` peer dependencies would be updated to the latest version as well and their version range would be extended to at least include minor releases (`^1.0.0`).

### Angular version

13.2.0

### Taiga UI version

2.31.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #1274 

## What is the new behavior?

Use npm 7 for correct resolve dependencies in monorepo

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/addon-doc/package.json [hunk 1]
```diff
@@ -7,23 +7,24 @@
...
+    },
     "peerDependencies": {
         "@angular/common": ">=9.0.0",
         "@angular/core": ">=9.0.0",
         "@angular/forms": ">=9.0.0",
         "@angular/router": ">=9.0.0",
-        "@angular-devkit/schematics": "^9.0.0",
         "@ng-web-apis/common": "1.12.1",
         "@tinkoff/ng-polymorpheus": "3.1.12",
         "@taiga-ui/cdk": "2.31.0",
```
Comment (hakimio) [inline projects/addon-doc/package.json:27]: Can you either extend "@ng-web-apis/common" and "@tinkoff/ng-polymorpheus" library version range or move them to "hard dependencies"?

Comment (splincode) [reply projects/addon-doc/package.json:27]: I can add like this in peerDependencies

```json
 "@tinkoff/ng-polymorpheus": "^3.1.12",
 "@taiga-ui/cdk": "^2.31.0",
```

but I'm not sure if adding in hard dependencies is the right way

Comment (hakimio) [reply projects/addon-doc/package.json:27]: Yeah, that solution could work for `@tinkoff/ng-polymorpheus` and `@taiga-ui/cdk`, but it doesn't work for `@ng-web-apis/common` which is already at version 2.

Comment (hakimio) [reply projects/addon-doc/package.json:27]: Looks good to me 👍

**Pull request closed without merging**